### ISSUE TITLE
able to collect logs/metrics from EventHub to Splunk but still getting an error message

### ISSUE THREAD

**Issue by GGGName** [microsoft/AzureMonitorAddonForSplunk#81]
I'm able to collect all data to Splunk, but still getting an error message when querying the following:
index="_internal" error

I'm getting this error:
09-28-2018 11:04:59.480 +0000 ERROR ExecProcessor - message from "python /opt/splunk/etc/apps/TA-Azure_Monitor/bin/azure_monitor_metrics.py" get_metrics_to_get: Error returned from get_arm: 400 Client Error: Parentheses Error: only name and aggregationType set of parentheses are allowed: () and startTime eq 2018-09-28T11:03:48Z and endTime eq 2018-09-28T11:04:48Z

couldn't find how to make it disappear, any Idea?

**Comment by sebastus** [comment]
Just a thought. What exactly did you put in for tags on the resources to be monitored?

**Comment by sebastus** [comment]
It should say Metrics:ActiveConnections

**Comment by sebastus** [comment]
@GGGName is this still giving you trouble? What is your status?

**Comment by GGGName** [comment]
Yes, still same error.

**Comment by sebastus** [comment]
Just attempted to repro. Changed Metrics:* to Metrics:ActiveConnections for my event hub. I don't see any errors in the log; just warnings that I don't have the right sourceType in sourcetypes.json. I'm using v1.3.1 of the add-on with a Splunk Enterprise 7.0.4 vm based on the Azure Marketplace ARM template. I installed the VM, then the add-on, then added creds for metrics data input. No other changes to the environment.
